app/src/modules/settings/routes/data-model/field-detail/shared/related-collection-select.vue

```diff
@@ -18,14 +17,8 @@ const collectionExists = computed(() => {
 	return !!collectionsStore.getCollection(props.modelValue);
 });
 
-const availableCollections = computed(() => {
-	return orderBy(
-		collectionsStore.databaseCollections.filter((collection) => collection.meta),
-		['meta.sort', 'collection'],
-	);
-});
-
-const systemCollections = collectionsStore.crudSafeSystemCollections;
+const availableCollections = computed(() => collectionsStore.configuredCollections);
+const systemCollections = computed(() => collectionsStore.crudSafeSystemCollections);
```

Comment:
Do you know if these `computed(() => ...)` are needed? They are getters in a pinia store, and I'm not really sure about their reactiveness. So I opted to wrap them in computed, but we had it as non-computed value before.

Reply:
Not required, as long as it is not destructured [^1] 👌 

[^1]: https://pinia.vuejs.org/core-concepts/#Using-the-store